src/Lecture3.hs

```diff
@@ -60,7 +60,15 @@ of a weekday.
 >>> toShortString Monday
 "Mon"
 -}
-toShortString = error "TODO"
+toShortString :: Weekday -> String
+toShortString day = case day of
+  Monday -> "Mon"
+  Tuesday -> "Tue"
+  Wednesday -> "Wed"
+  Thursday -> "Thu"
+  Friday -> "Fri"
+  Saturday -> "Sat"
+  Sunday -> "Sun" 
```

Comment:
This is the correct implementation, however, it's a bit verbose. Try using standard functions `show` and `take` to implement it shortly 🙂